src/index.css

```diff
@@ -4,4 +4,11 @@
     --main-bg-color: #1e2024;
     --btn-bg-color: #34343d;
     --btn-text-color: #fff;
+    --b3f: 250, 250, 250
+}
+
+body {
+    margin: 0;
+    /* background-color: #151e28; */

```

Comment:
Here you can remove the comment if not required

Reply:
Done